### ISSUE TITLE
Bundles show "Snoozed x ago" when first item was snoozed

### ISSUE THREAD

**Issue by samuelalexmclean** [teresa-ou/inboxy#76]
Sometimes, when I see a bundle, it will say "Snoozed x days ago"

![Photoshop_tuntMZ8Tse](https://user-images.githubusercontent.com/10862750/97821855-fb1eac00-1d07-11eb-9a6b-cbe0e3c90309.png)

When I click into the bundle, that's only true of the first item:

![Photoshop_AMhGXfWeRJ](https://user-images.githubusercontent.com/10862750/97821851-f9ed7f00-1d07-11eb-85a0-4145e9de8647.png)

At first brush this is to be expected - usually, a bundle would display the date of the most recent activity. Since the date in this case has been supplanted by "Snoozed x days ago" that gets shown instead.

I think the problem here is that absolute dates (whether on individual emails, threads or bundles) are always representing the last update time, which is true for the entire entity. Whereas the snooze recency applies only to one item within the bundle - but it's being presented at the top level, which doesn't really make sense.

I think it would make more sense to display the max date from within the bundle. This gives you a true idea of when that bundle was last modified, and you can always click into it to see the snooze info. WDYT?

**Comment by teresa-ou** [comment]
> I think the problem here is that absolute dates (whether on individual emails, threads or bundles) are always representing the last update time, which is true for the entire entity. Whereas the snooze recency applies only to one item within the bundle - but it's being presented at the top level, which doesn't really make sense.

Yeah, that's a good point. I agree the max date from within the bundle would be more representative, although then when a snoozed message in the bundle reappears in your inbox, you wouldn't see it until you open the bundle. So I'm thinking the ideal solution would be to use the max date as you suggested, plus find a better way to indicate that a snoozed message has reappeared.

**Comment by leggett** [comment]
FWIW, I'm pretty sure the way this worked in Google Inbox was for the snoozed message show outside the bundle when it reappeared. I'm not certain, but I think that is how it worked. Not to say that is how it should work here...

**Comment by benoror** [comment]
Hi! Showing "Snoozed …" on the collapsed bundle from only the first thread is still a known rough edge. Not specifically fixed in **Inbundly** yet.

Please open a bug if you still hit it. Related new feature: **snooze whole bundle** intentionally applies Gmail snooze to every thread in the bundle.

Chrome: https://chromewebstore.google.com/detail/inbundly/pbfjicjmcpogjlbpljebhhgkgfhbdcga
Issues: https://github.com/benoror/inbundly/issues/new
